packages/otel/src/otel-wrapper.ts

```diff
@@ -21,7 +20,7 @@ if (process.env.OTEL_ENABLED === "true" && process.env.OTEL_SERVICE_NAME) {
 
 const OTEL_FLUSH_TIMEOUT = sharedOtelConfig.flushTimeout;
 
-const flushOtel = async () => {
+export const flushOtel = async () => {
```

Comment:
lets move it to separate file. you import this from migration scripts, where you start otel SDK. But you actually start it here too (L18-19) (side effect)

Reply:
Removed export - we don't need to flush otel for migrations